api/rbdigital.py

```diff
@@ -801,14 +822,19 @@ def get_patron_checkouts(self, patron_id):
 
         # by now we can assume response is either empty or a list
         for item in resp_obj:
-            loan_info = self._make_loan_info(item)
+            loan_info = self._make_loan_info(
+                item, fulfill_part_url=fulfill_part_url
+            )
             if loan_info:
                 loans.append(loan_info)
         return loans
 
-    def _make_loan_info(self, item, fulfill=False):
+    def _make_loan_info(self, item, fulfill_part_url=None):
```

Comment:
The old `fulfill` argument wasn't being used at all. Despite appearances, `fulfill` is not being replaced by something similar; I just removed `fulfill` because it didn't do anything.